charts/kong/example-values/doc-examples/quickstart-enterprise-licensed-aio.yaml

```diff
@@ -151,7 +151,7 @@ extraLabels:
   konghq.com/component: quickstart
 image:
   repository: kong/kong-gateway
-  tag: "2.8"
+  tag: "3.0"
```

Comment:
🤔 Which registry + repo actually do we mean here? This doesn't seem to exist under: https://hub.docker.com/kong/kong-gateway

Reply:
![image](https://user-images.githubusercontent.com/59130/189680561-c6078e2f-daac-47f0-b37b-c1c35c7fb50f.png)

It exists for me

Reply:
<img width="851" alt="image" src="https://user-images.githubusercontent.com/739996/189681268-08bd6057-4250-49f6-8826-3e7f17cb2cf7.png">

😿

Reply:
Ah, it's a public image but can't be listed publicly. `docker pull kong/kong-gateway:3.0` should work fine for you

Reply:
👍